src/common/snippets/src/lowered/pass/validate_loops.cpp

```diff
@@ -63,8 +63,6 @@ bool ValidateLoops::run(LinearIR& linear_ir) {
                                     "Incorrect Loop ID configuration: the Loops with splitted dimension should be successively nested");
                     OPENVINO_ASSERT(loop_manager->get_loop_info(loop_ids[i - 1])->get_increment() == loop_manager->get_loop_info(id)->get_work_amount(),
                                     "Incorrect Loop ID configuration: the Loops with splitted dimension should be successively nested");
-                    OPENVINO_ASSERT(loop_manager->get_loop_info(loop_ids[i - 1])->get_outer_splited_loop(),
```

Comment:
Could you please clarify why you removed this check as well?

Reply:
The check was removed because `m_outer_splited_loop` flag was removed